index.d.ts

```diff
@@ -27,7 +27,7 @@ export type CommonOptions<EncodingType> = {
 
 	If you `$ npm install foo`, you can then `execa('foo')`.
 
-	@default `true` with `$`/`$.sync`, `false` otherwise
```

Comment:
I think `$.sync` is implied by `$`, so I am not sure we need to be explicit there.